client/src/components/syncfusion/Scheduler.tsx

```diff
@@ -270,9 +270,13 @@ const QuickInfoTemplatesContent: any = ({
           />
         </>
       ))}
-    <div>
-      <h3>Created by {fullName}</h3>
-    </div>
+    {!Id ? (
+      <div></div>
+    ) : (
+      <div>
+        <h3>Created by {fullName}</h3>
+      </div>
+    )}
```

Comment:
Suggestion:
```
{!Id && (
    // Created by stuff
)}
```
This makes it so that we don't have to define two cases and have an unnecessary div. If `Id` has a value, the `&&` returns false, telling react to ignore this block